lib/manageiq/appliance_console/cli.rb

```diff
@@ -273,12 +273,12 @@ def run
       saml_unconfig if saml_unconfig?
       oidc_config if oidc_config?
       oidc_unconfig if oidc_unconfig?
-      set_server_state if set_server_state?
       openscap if openscap?
       message_server_config if message_server_config?
       message_server_unconfig if message_server_unconfig?
       message_client_config if message_client_config?
       message_client_unconfig if message_client_unconfig?
+      set_server_state if set_server_state?
```

Comment:
NOTE moving this after the `message_*_config` so that you can configure db and/or messaging and also start the server

Reply:
Can you add that as a literal comment so someone doesn't come along in the future and just reorder them (say, for like alphabetical reasons)?  Asking because I don't think we have specs here.

Reply:
:+1: done